content/changelog-stable/index.html

```diff
@@ -55,6 +55,55 @@
   <div id="rc" style="display:none;">
   </div>
 
+<h3><a name=v2.7.3>What's new in 2.7.3</a> (2016/08/31)</h3>
+<ul class=image>
+
+  <li class="major bug">
+    Stop A/B testing of the remoting JNLP3 protocol due to the known issues.
+    The protocol can be enabled manually via the <code>jenkins.slaves.JnlpSlaveAgentProtocol3.enabled</code>
+    <a href="https://wiki.jenkins-ci.org/display/JENKINS/Features+controlled+by+system+properties">system property</a>.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-37315">issue 37315</a>)
+  <li class="bug">
+    When checking Update Center, append <code>?uctest</code> parameter to HTTP and HTTPS URLs only.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-37189">issue 37189</a>)
+  <li class="bug">
+    Ensure that detached plugins are always at least their minimum version.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-37041">issue 37041</a>)
+  <li class="bug">
+    Remove trailing space from <tt>Hudson.DisplayName</tt> in Spanish, which resulted in problems with Blue Ocean.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-36940">issue 36940</a>)
+  <li class="bug">
+    Make sure that the All view is created.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-36908">issue 36908</a>)
+  <li class="bug">
+    Incorrect formatting of messages in the Upate Center and Setup Wizard.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-36757">issue 36757</a>)
+  <li class=bug>
+    Underprivileged users were unable to use the default value of a password parameter.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-36476">issue 36476</a>)
+  <li class=bug>
+    Properly handle exceptions during global configuration form submissions when <code>SCM Retry Count</code> field is empty.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-36387">issue 36387</a>)
+  <li class='bug'>
+    Do not allow disabled project to be triggered remotelly.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-36193">issue 36193</a>)
+  <li class=bug>
+    Ensure that <code>SCMDescriptor.newInstance</code> overrides are honored when creating new SCM entries.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-36043">issue 36043</a>,
+     <a href="https://issues.jenkins-ci.org/browse/JENKINS-35906">issue 35906</a>
+    , regression in 2.10)
```

Comment:
It is since we have backported the commit that broke it in JENKINS-35098. The note "regression in 2.10" is irrelevant.